insights_content_template_renderer/tests/test_dot.py

```
@@ -23,13 +22,17 @@ def test_single_quote_correct_handling():
 
 def test_nonletter_characters_correct_handling():
     """
-    Checks that DoT.template function does not remove characters like
+    Checks that dot.template function does not remove characters like
     escape characers and single quotes from the given input if strip is
     set to False
     """
-    input = r"Red Hat recommends that you configure your nodes to meet the minimum resource requirements.\n\nMake " \
-            r"sure that:\n\n1. Node foo1 (undefined) * Has enough memory, minimum requirement is 16. Currently its " \
-            r"only configured with 8.16GB. "
+    input = (
```

Ruff (0.15.6)

[error] Variable `input` is shadowing a Python builtin

(A001)